streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -331,6 +344,15 @@ private void assertNextOutputRecord(String topic, String key, String value, Inte
         assertEquals(partition, record.partition());
     }
 
+    private void assertNextOutputRecord(String topic, String key, String value, Integer partition, Long timestamp) {
```

Comment:
@sebigavril I know that 2 of the `assertNextOutputRecord` methods were already there, but if there is some way for us to reduce the duplicate logic that would be great.  Unfortunately, I don't have any good ideas ATM.